### PR TITLE
Add robot Bulldog

### PR DESCRIPTION
**Pull request by wzy0426**
Add new robot "Bulldog"

### CODE REVIEW

File: add_robot.py~ [hunk 1]
```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
```
Comment (sloretz) [inline add_robot.py~:1]: It looks like this file might have been accidentally committed. Would you mind removing it from the pull request?

File: _posts/2017-07-21-bulldog.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+* Rating&nbsp; &nbsp; &nbsp;&nbsp; &nbsp;&nbsp; &nbsp; &nbsp;&nbsp; &nbsp;&nbsp; &nbsp; &nbsp;&nbsp; &nbsp;&nbsp; &nbsp; &nbsp;&nbsp; &nbsp;&nbsp; &nbsp; &nbsp;&nbsp; &nbsp;&nbsp; &nbsp; &nbsp;&nbsp; &nbsp;&nbsp;&nbsp; IP66
+* Encoder&nbsp;&nbsp; &nbsp; &nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp;&nbsp; 25000Pulses/m
+* Feedback &nbsp; &nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp;&nbsp; battery status/running status
+* Control Mode&nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp;&nbsp; Voltage,Speed,Whell,Speed
+* Communitcation&nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp; &nbsp;USB/Ethernet
```
Comment (sloretz) [inline _posts/2017-07-21-bulldog.md:43]: I'm not sure a format built from non-breaking spaces will display right on all platforms. An alternative is sub bullets, those can be done with two spaces before the asterisk.

```
* Dimensions (L*W*H)
  * 880x680x450m
```
 becomes
* Dimensions (L*W*H)
  * 880x680x450m

File: _posts/2017-07-21-bulldog.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+main-class: 'ground'
+color: '#2828FF'
+tags:
+- 'ground'
+- 'mobile robot'
+- 'outdoor'
+- 'teleoperator'
+twitter_text: Checkout the Bulldog
+website: http://intelrope.de/category/products/mobile-robots-outdoor
+wiki_homepage: http://wiki.ros.org/Robots/Bulldog
```
Comment (sloretz) [inline _posts/2017-07-21-bulldog.md:18]: I notice this wiki page doesn't exist yet. Will it in the future?

File: _posts/2017-07-21-bulldog.md~ [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (sloretz) [inline _posts/2017-07-21-bulldog.md~:1]: Was this file accidentally committed?

File: add_robot.py~ (hunk 1 above)
Comment (wzy0426) [inline add_robot.py~]: no,You can remove it! Thank you.

File: _posts/2017-07-21-bulldog.md (hunk 3 above)
Comment (wzy0426) [inline _posts/2017-07-21-bulldog.md]: Yes,We are preparing this.

File: _posts/2017-07-21-bulldog.md~ (hunk 4 above)
Comment (wzy0426) [inline _posts/2017-07-21-bulldog.md~]: Yes,you can remove it also.I dont't know much about.

File: _posts/2017-07-21-bulldog.md (hunk 2 above)
Comment (wzy0426) [inline _posts/2017-07-21-bulldog.md]: I have tested it in locally by docker.It's no problem.I worked as an intern in a robot company.This is a  
task from my boss.Can this "Bulldog"  be display on wiki ? Is there anything else in need to change?
Thank you.

File: add_robot.py~ (hunk 1 above)
Comment (wzy0426) [reply add_robot.py~:1]: no，i will remove it ,Thanks for you reply.

File: _posts/2017-07-21-bulldog.md (hunk 3 above)
Comment (wzy0426) [reply _posts/2017-07-21-bulldog.md:18]: yes,we are preparing this.

**Pull request closed without merging**